h2o-algos/src/main/java/hex/tree/DTree.java

```diff
@@ -1108,14 +1232,12 @@ static Split findBestSplitPoint(DHistogram hs, int col, double min_rows,
       return null;
     }
 
-    final double node_p0 = predLeft / nLeft;
-    final double node_p1 = predRight / nRight;
-
     // FIXME (PUBDEV-7553): these asserts do not hold because histogram doesn't skip rows with NA response
     // assert hasNomin || nomLeft == predLeft;
```

Comment:
would the uncommented code still work?

Reply:
My code does not affect `nomLeft` and `nomRight` values. So it should work.